src/PactNet/Drivers/HttpInteractionDriver.cs

```diff
@@ -37,8 +37,11 @@ public void Given(string description)
         /// <param name="name">Parameter name</param>
         /// <param name="value">Parameter value</param>
         /// <returns>Success</returns>
-        public void GivenWithParam(string description, string name, string value)
-            => NativeInterop.GivenWithParam(this.interaction, description, name, value).CheckInteropSuccess();
+        public void GivenWithParam(string description, string name, object value)
+            {
+                var jsonValue = System.Text.Json.JsonSerializer.Serialize(value);
```

Comment:
style: Don't use var when it's not obvious what the type is 

style: Don't use fully qualified references inline, use an import instead